src/components/viewContainer.js

```diff
@@ -1,20 +1,19 @@
-define(['browser', 'dom', 'layoutManager', 'css!components/viewManager/viewContainer'], function (browser, dom, layoutManager) {
-    'use strict';
+/* eslint-disable indent */
```

Comment:
where is `css!components/viewManager/viewContainer`?

Reply:
[here?](https://github.com/jellyfin/jellyfin-web/blob/master/src/components/viewManager/viewContainer.css)

Reply:
I mean where is its `import`

Reply:
Oh right 🤦‍♀️

Reply:
Indeed, I was a bit overzealous when removing unused imports :p